### ISSUE TITLE
Terminus Command: Backup get -> Example usage section typo

### ISSUE THREAD

**Issue by yuri-pochepko** [pantheon-systems/documentation#6133]
Re: https://pantheon.io/docs/terminus/commands/backup-get

Priority: Low

## Issue Description

Duplicated block with typo in Example usage section.

<img width="891" alt="example usage" src="https://user-images.githubusercontent.com/63350179/100251525-ecff2c80-2f4f-11eb-953f-a8cb88aa5b87.png">

## Suggested Resolution

`backup:get <site>.<env> --element=<element> --to=<path> Saves the most recent <element> backup in <site>'s <env> environment to <path>.`

**Comment by yuri-pochepko** [comment]
Pull request into Terminus https://github.com/pantheon-systems/terminus/pull/2103

**Comment by alexfornuto** [comment]
Thanks @yuri-pochepko! As you know based on your PR to Terminus, this content is pulled from the app directly. Once that PR is merged into Terminus and released, the docs can (and will) be updated.

Since I can't do anything about it until then, and we already track and update for new Terminus releases, I'm going to close this issue.
